### PR TITLE
Instruction to change tor browser security slider updated

### LINKED ISSUE #2962: Instructions to change Tor Browser security slider need updating
## Description

We have instructions to turn the security slider to its maximum setting on the source interface. These instructions are out of date with the security slider. 

## Steps to Reproduce

0. Access the source interface in latest Tor Browser (here Tor Browser 7.5)
1. Try to follow the instructions to turn the security slider to "High":

screen shot 2018-02-02 at 4 20 43 pm

## Expected Behavior

I follow instructions as written and I am not confused

## Actual Behavior

The highest setting is now called "Safest":

screen shot 2018-02-02 at 4 24 06 pm

## Comments

Just replace "High" with "Safest"

### PR DESCRIPTION
**Pull request by Anonymous26**
## Status

Ready for review 

## Description of Changes

Fixes #2962 

Changes proposed in this pull request: Updated the instructions to change the security in tor browser

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made changes to documentation:

- [x] Doc linting passed locally

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -260,9 +260,9 @@ via the Ansible config at install time (#1620)
 
 ## 0.3.11
 
-* Instructs source to turn the Tor Browser security slider to High and to
+* Instructs source to turn the Tor Browser security slider to Safest and to
```
Comment (dachary) [inline changelog.md]: There is no need to change High to Safest here: the release notes were published at a time when that was true. We do not rewrite history :-)

**Comment by dachary** [comment]
I see you have added a screenshot 👍 Please let me know if updating images/manual/source-turn-slider-to-high.png gives you trouble.

**Comment by Anonymous26** [comment]
@dachary  Updated rest of the screenshots

**Comment by Anonymous26** [comment]
@dachary Is anything left here ?

**Comment by dachary** [comment]
@Anonymous26 this looks good, thanks for the ping. Could you please resize the images so their size is close to the size of the image they replace ?

**Comment by Anonymous26** [comment]
Sure

**Comment by dachary** [comment]
also docs/images/manual/source-turn-slider-to-high.png shows "turn the Security Slider to High" instead of "turn the Security Slider to Safest"

**Comment by redshiftzero** [comment]
@Anonymous26 it looks like we just need `docs/images/manual/source-turn-slider-to-high.png` updated and then this will be good to merge :sparkles:

**Comment by Anonymous26** [comment]
ill fix this ASAP

**Comment by Anonymous26** [comment]
@dachary  I've updated the pr accordingly pls review.

**Pull request merged**